phpunit-api-coverage.xml

```diff
@@ -13,4 +11,8 @@
             <directory suffix=".php">src</directory>
         </whitelist>
     </filter>
+
+    <php>
+        <const name="SDK_TESTING_ENV" value="api_coverage" />
```

Comment:
I didn't actually think about it, to be honest.
Do you mean simply changing this to use `<env>` instead of `<const>` ([docs](https://phpunit.de/manual/current/en/appendixes.configuration.html#appendixes.configuration.php-ini-constants-variables)), or actually having environment variables set in whatever thing is running the coverage tool?